env/env.go

```diff
@@ -46,7 +46,7 @@ func (e Environment) String() string {
 // is required.
 func New() (Environment, error) {
 	e := Environment{
-		RepoToken: findVar([]string{"CC_REPO_TOKEN", "REPO_TOKEN"}),
+		RepoToken: findVar([]string{"CC_REPO_TOKEN", "REPO_TOKEN", "CC_TEST_REPORTER_ID"}),
```

Comment:
You can remove the first two elements. We'll only look at `CC_TEST_REPORTER_ID` in the new client -- clean break!